### PR TITLE
Minor style fixes

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Dlna/Didl/DidlBuilder.cs [hunk 1]
```diff
@@ -392,24 +392,24 @@ private string GetDisplayName(BaseItem item, StubType? itemStubType, BaseItem co
...
-                    case StubType.Playlists:        return _localization.GetLocalizedString("Playlists");
-                    case StubType.AlbumArtists:     return _localization.GetLocalizedString("HeaderAlbumArtists");
-                    case StubType.Albums:           return _localization.GetLocalizedString("Albums");
-                    case StubType.Artists:          return _localization.GetLocalizedString("Artists");
-                    case StubType.Songs:            return _localization.GetLocalizedString("Songs");
-                    case StubType.Genres:           return _localization.GetLocalizedString("Genres");
-                    case StubType.FavoriteAlbums:   return _localization.GetLocalizedString("HeaderFavoriteAlbums");
-                    case StubType.FavoriteArtists:  return _localization.GetLocalizedString("HeaderFavoriteArtists");
-                    case StubType.FavoriteSongs:    return _localization.GetLocalizedString("HeaderFavoriteSongs");
+                    case StubType.Latest: return _localization.GetLocalizedString("Latest");
```
Comment (JustAMan) [inline Emby.Dlna/Didl/DidlBuilder.cs]: I actually liked old style with visual alignment, why remove?

File: Emby.Dlna/Main/DlnaEntryPoint.cs [hunk 2]
```diff
@@ -262,8 +262,8 @@ private async Task RegisterServerEndpoints()
             {
                 if (address.AddressFamily == AddressFamily.InterNetworkV6)
                 {
-                   // Not support IPv6 right now
-                   continue;
+                    // Not support IPv6 right now
```
Comment (JustAMan) [inline Emby.Dlna/Main/DlnaEntryPoint.cs]:
```suggestion
                    // Not supporting IPv6 right now
```

File: RSSDP/DisposableManagedObjectBase.cs [hunk 3]
```diff
@@ -72,7 +72,7 @@ public string BuildMessage(string header, Dictionary<string, string> values)
         /// <para>Sets the <see cref="IsDisposed"/> property to true. Does not explicitly throw an exception if called multiple times, but makes no promises about behaviour of derived classes.</para>
         /// </remarks>
         /// <seealso cref="IsDisposed"/>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1063:ImplementIDisposableCorrectly", Justification="We do exactly as asked, but CA doesn't seem to like us also setting the IsDisposed property. Too bad, it's a good idea and shouldn't cause an exception or anything likely to interfer with the dispose process.")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1063:ImplementIDisposableCorrectly", Justification = "We do exactly as asked, but CA doesn't seem to like us also setting the IsDisposed property. Too bad, it's a good idea and shouldn't cause an exception or anything likely to interfer with the dispose process.")]
```
Comment (JustAMan) [inline RSSDP/DisposableManagedObjectBase.cs]: Is this suppression valid? I mean, it seems to turn off a warning, maybe that warning actually makes sense to be fixed?

File: Emby.Dlna/Didl/DidlBuilder.cs (hunk 1 above)
Comment (Bond-009) [reply Emby.Dlna/Didl/DidlBuilder.cs:395]: This whole PR was done by ReSharper, not me

File: RSSDP/DisposableManagedObjectBase.cs (hunk 3 above)
Comment (Bond-009) [reply RSSDP/DisposableManagedObjectBase.cs:75]: It isn't, but this is in the RSSDP repo, that whole thing is a mess imo

**Pull request merged**